### PR TITLE
Cleanup/move model definitions to unity

### PR DESCRIPTION
**Pull request by bendecoste**

### CODE REVIEW

File: UnityProject/Assets/OpenMined/Syft/Model/Linear.cs [hunk 1]
```diff
@@ -1,11 +1,39 @@
-﻿using System;
+﻿using JetBrains.Annotations;
```
Comment (bosr) [inline UnityProject/Assets/OpenMined/Syft/Model/Linear.cs]: Are you sure about this line? (just asking, this is used nowhere else for the moment)

File: UnityProject/Assets/OpenMined/Syft/Model/Model.cs [hunk 2]
```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Collections.Generic;
+using JetBrains.Annotations;
```
Comment (bosr) [inline UnityProject/Assets/OpenMined/Syft/Model/Model.cs]: same remark ;)

**Comment by bendecoste** [comment]
Good call .. I use rider so the imports get done for me ... I’ll verify I
didn’t break mono develop

**Comment by iamtrask** [comment]
Great job @bendecoste for implementing and @bosr for attention to detail!!

**Pull request merged**